controls/radwordsprocessing/editing/insert-documents.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Insert Documents
+page_title: Insert Documents
+description: Insert Documents
+slug: radwordsprocessing-editing-insert-documents
+tags: insert, document, position
+published: True
+position: 1
+---
+
+# Insert Documents
+
+With **RadWordsProcessing**, you have the ability to insert a document into another document at specified position. 
+
+## Inserting documents at a specific position
```

Comment:
The casing of the title should be modified:  Inserting Documents at a Specific Position